Library/List/EntitlementList.cs

```diff
@@ -0,0 +1,49 @@
+using System;
+using System.Collections.Specialized;
+using System.Net;
+using System.Web;
+using System.Xml;
+
+namespace Recurly
+{
+    public class EntitlementList : RecurlyList<Entitlement>
+    {
+        internal EntitlementList(string baseUrl) : base(Client.HttpRequestMethod.Get, baseUrl)
```

Comment:
i dont think this needs to inherit from client

Reply:
I was following what we do on NoteList and BillingInfoList and without this I don't get my entitlements array

Reply:
ya i see now, it is required